src/mbgl/renderer/renderer_impl.cpp

```diff
@@ -258,19 +258,6 @@ void Renderer::Impl::render(const UpdateParameters& updateParameters) {
         staticData = std::make_unique<RenderStaticData>(backend.getContext(), pixelRatio, programCacheDir);
     }
 
-    PaintParameters parameters {
```

Comment:
I guess that would clash with https://github.com/mapbox/mapbox-gl-native/pull/14335 ...